content/developer-platform/platform-users/consuming-services/_index.en.md

```diff
@@ -32,13 +31,18 @@ Note that every Service shows:
   manually enable the service using `kubectl`)
 * a short description
 
+Check out [Your First Service]({{< relref "../../tutorials/your-first-service/" >}}) if you want to publish services yourself.
+
 ## Enabling a Service
 
-Before a KPD Service can be used, it must be enabled in the workspace where it should be available.
+After clicking on "Add to Organization" in the service catalog, the following wizard will open to confirm adding a service:
+
+![Add Service Wizard](add-service-wizard.png?classes=shadow,border&height=200 "Wizard for adding a service to the workspace, showing options for creating default rules and resource claims")
 
-### Dashboard
+Several options are offered when adding a service:
 
-(TODO: currently the UI has no support for this.)
+- "Create default roles for resources managed by this service" will optionally create a standard set of RBAC roles within the workspace that allow giving different levels of access to this specific service. This is not mandatory, but otherwise roles need to be created yourself.
+- The "Resources" section informs you about resources that the service is requesting access (claiming) to function properly, e.g. to be able to read or write secret data associated with objects of that service. Service owners are encouraged to gracefully handle situations in which resource access is not granted, but service quality and functionality will likely suffer.
```

Comment:
You could add "you can grant/revoke access to resources for a service at any time later" or similar.

Reply:
:+1: